tach/cache/setup.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import os
+import uuid
+from pathlib import Path
+
+
+def resolve_project_dir() -> Path:
+    current_dir = os.getcwd()
```

Comment:
This should work:

```python
from pathlib import Path

current_dir = Path.cwd()
tach_path = current_dir / "tach.yml"
```

`pathlib` basically wraps everything you'd want to use in `os`.